.github/workflows/node-build.yml

```diff
@@ -69,6 +81,9 @@ jobs:
     timeout-minutes: 5
```

Comment:
for `token-introspection` we need to update how we get the specfications into the project. Currently, in `token-instrospection/package.json` we copy over the specs from the NodeSDK, and then use a symlink into `token-introspection/src/openapi`. Instead, we should use the specs from the submodule